package.json

```diff
@@ -2,13 +2,16 @@
   "name": "backpat",
   "version": "0.5.0",
   "description": "A simple tool for high-fiving your tech stack",
-  "main": "src/index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "test": "nyc mocha",
+    "prebuild": "rimraf dist",
```

Comment:
Any script in package.json beginning with the "pre" prefix, will run before its non-pre counterpart is run. So, in this case, prebuild will always run before "build" is run.

Essentially, this is saying before the production build is run, remove all of the files in the dist directory. We need to use rimraf because directory removal is not uniform across platforms (i.e. _rm_ _-rf_ _dist_ on Macs and _rmdir_ _/s_ _dist_ on Windows.)